README.md

```diff
@@ -1,16 +1,27 @@
-# Angular product template
+# UiComponentsAngularTemplate
 
-To help get teams up and running sooner, this product template to DDI product teams to showcase the ui-components and provide a starting point for teams building DDI products.
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 14.0.5.
 
-## Get started
+## Development server
 
-To get started with Angular click the `Use this template` button.
+Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
 
-## Storybook docs
+## Code scaffolding
 
-Documentation for all available components can be viewed at [Storybook: Angular UI Components](https://ui-components-ui-components-dev.os99.gov.ab.ca/)
+Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
 
+## Build
 
-## Bugs
+Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory.
 
-Any issues with this template, or the Angular components, should be created at [Bugs: Angular UI Components](https://github.com/GovAlta/ui-components/issues/new?assignees=&labels=bug&template=log-a-bug.md&title=)
+## Running unit tests
+
+Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+
+## Running end-to-end tests
+
+Run `ng e2e` to execute the end-to-end tests via a platform of your choice. To use this command, you need to first add a package that implements end-to-end testing capabilities.
+
+## Further help
+
+To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
```

Comment:
Can you remove these changes? We need to keep the previous README file

Reply:
done